cves/kernel/CVE-2016-3137.yml

```diff
@@ -377,16 +376,16 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
-lessons:
+  answer: true
+  note: 'Part of the fix involved moving the check for the existance of the interrupt in and out to another portion of the code so the check happened sooner.'
```

Comment:
a little confused by what you mean by "in and out to another portion."

Reply:
I meant it as interrupt in and interrupt out but I can see how that is misleading.